reformat_gherkin/formatter.py

```diff
@@ -113,6 +114,26 @@ def extract_description_lines(description: Optional[str]) -> List[str]:
     return description.splitlines()
 
 
+def get_display_width(text: str) -> int:
```

Comment:
This function seems to be a more generic function, which should be placed in `utils.py`

Reply:
Ok, I've moved the function to utils.py and the tests to test_utils.py.